server/service/Cargo.toml

```diff
@@ -30,7 +30,7 @@ serde_yaml = "0.8.24"
 schemafy = "0.6.0"
 schemafy_core = "0.6.0"
 tera = "1"
-tokio = { version = "1.17.0", features = ["macros", "sync", "time"] }
+tokio = { version = "1.17.0", features = ["macros", "sync", "time", "fs"] }
```

Comment:
This is a fix for a tokio error I was getting whenever I tried to run tests. @andreievg 's suggestion.